tests/foreman/ui/test_packages.py

```diff
@@ -123,3 +131,68 @@ def test_positive_check_package_details(self):
                     ['Build Time', '1331831364'],
                 ]
             )
+
+
+class RHPackagesTestCase(UITestCase):
+    """Implement tests for packages via UI"""
+
+    @classmethod
+    @skip_if_not_set('fake_manifest')
+    def setUpClass(cls):
+        super(RHPackagesTestCase, cls).setUpClass()
+        cls.organization = entities.Organization().create()
+        with manifests.clone() as manifest:
+            upload_manifest(cls.organization.id, manifest.content)
+        repo_id = enable_rhrepo_and_fetchid(
+            basearch='x86_64',
+            org_id=cls.organization.id,
+            product=PRDS['rhel'],
+            repo=REPOS['rhst7']['name'],
+            reposet=REPOSET['rhst7'],
+            releasever=None,
+        )
+        entities.Repository(id=repo_id).sync()
+
+    @tier3
```

Comment:
git grep for `upload_manifest(` phrase and take a look on tiers - i'd say 60% of tests are marked as tier2, 40% as tier1 and i can't see any test marked as tier3 just because of subscription upload, pretty much the same picture is for repository sync. Tier3 tests are system tests and we aren't testing system here but checking field values and mentioned preconditions never rose tier for other tests. Without preconditions i can't even see a point of why it could be tier3 - for content search functionality verification that search returns proper results is definitely tier1, that's the most basic functionality we can test.